eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -45,15 +45,15 @@ library
     build-depends:
         base >=4.14 && <4.19,
         bytestring,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.11,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} ^>=1.6,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib} ^>=1.3,
+        cardano-ledger-core:{cardano-ledger-core, testlib} ^>=1.10,
```

No changes to the package
```suggestion
        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.11,
```